### PR TITLE
Port SignedRPMs documentation page to Markdown.

### PR DESCRIPTION
**Pull request by bbockelm**
Note that this also enables the codehilite extension, allowing us to mark blocks of text as console output.

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SignedRPMS

- [ ] Add migration header to TWiki document

### CODE REVIEW

File: docs/Release/signing.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+For example:
+
+```console
+$ rpm --checksig -v globus-core-8.0-2.osg.x86_64.rpm 
+globus-core-8.0-2.osg.x86_64.rpm:
+    Header V3 DSA signature: OK, key ID 824b8603
+    Header SHA1 digest: OK (2b5af4348c548c27f10e2e47e1ec80500c4f85d7)
+    MD5 digest: OK (d11503a229a1a0e02262034efe0f7e46)
+    V3 DSA signature: OK, key ID 824b8603
+```
```
Comment (brianhlin) [inline docs/Release/signing.md]: I'm not sure if it's a codehilite issue but parts of this seem arbitrarily highlighted:

- `-v`
- `-core-8.0-2.`
- `824`

File: docs/Release/signing.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| Key ID       | `824b8603`                                             |
+
+You can see the fingerprint for yourself:
+
+```console
+$ gpg --with-fingerprint /etc/pki/rpm-gpg/RPM-GPG-KEY-OSG
+pub  1024D/824B8603 2011-09-15 OSG Software Team (RPM Signing Key for Koji Packages) <vdt-support@opensciencegrid.org>
+      Key fingerprint = 6459 D9D2 AAA9 AB67 A251  FB44 2110 B1C8 824B 8603
+sub  2048g/28E5857C 2011-09-15
+```
```
Comment (brianhlin) [inline docs/Release/signing.md]: Similar highlighting issues as above

File: docs/Release/signing.md (hunk 1 above)
Comment (bbockelm) [inline docs/Release/signing.md]: I think this might be your laptop?  It looks fine to me...

Comment (bbockelm) [inline docs/Release/signing.md]: Here's what I see:

codehilite_signing

If I don't have codehilite functioning, then I get the random arbitrary highlights you are complaining about.

How about we merge; we can revert if the issue also pops up with the version used by Travis?

Comment (brianhlin) [inline docs/Release/signing.md]: It still looks broken to me, with the numbers I mentioned having different classes:

```html
<pre class="codehilite"><code class="language-console hljs vbnet">$ gpg --<span class="hljs-keyword">with</span>-fingerprint /etc/pki/rpm-gpg/RPM-GPG-<span class="hljs-keyword">KEY</span>-OSG
pub  <span class="hljs-number">1024</span>D/<span class="hljs-number">824</span>B8603 <span class="hljs-number">2011</span>-<span class="hljs-number">09</span>-<span class="hljs-number">15</span> OSG Software Team (RPM Signing <span class="hljs-keyword">Key</span> <span class="hljs-keyword">for</span> Koji Packages) &lt;vdt-support@opensciencegrid.org&gt;
      <span class="hljs-keyword">Key</span> fingerprint = <span class="hljs-number">6459</span> D9D2 AAA9 AB67 A251  FB44 <span class="hljs-number">2110</span> B1C8 <span class="hljs-number">824</span>B <span class="hljs-number">8603</span>
<span class="hljs-keyword">sub</span>  <span class="hljs-number">2048</span>g/<span class="hljs-number">28E5857</span>C <span class="hljs-number">2011</span>-<span class="hljs-number">09</span>-<span class="hljs-number">15</span></code></pre>
```

**Comment by bbockelm** [comment]
Yeah - the code generated locally is something like this:
```html
<div class="codehilite"><pre><span class="gp">$</span> gpg --with-fingerprint /etc/pki/rpm-gpg/RPM-GPG-KEY-OSG
<span class="go">pub  1024D/824B8603 2011-09-15 OSG Software Team (RPM Signing Key for Koji Packages) &lt;vdt-support@opensciencegrid.org&gt;</span>
<span class="go">      Key fingerprint = 6459 D9D2 AAA9 AB67 A251  FB44 2110 B1C8 824B 8603</span>
<span class="go">sub  2048g/28E5857C 2011-09-15</span>
</pre></div>
```

I'm a bit puzzled as to why the `codehilite` extension isn't being invoked -- I think that's the underlying problem.

**Comment by bbockelm** [comment]
Ha!  It silently is disabled if `pygments` isn't installed.  Did the appropriate `pip install pygments` in the travis build and it now looks appropriate.

**Pull request merged**